### PR TITLE
Fixed logging info_keywords in the VecMonitor class.

### PR DESCRIPTION
**Pull request by ScheiklP**
## Description
closes https://github.com/DLR-RM/stable-baselines3/issues/728

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)

## To test
```
import gym
from stable_baselines3 import PPO
from stable_baselines3.common.vec_env import DummyVecEnv, VecMonitor
from gym.envs.registration import register
from stable_baselines3.common.envs.bit_flipping_env import BitFlippingEnv

register(
    id="BitFlippingEnv-v1",
    entry_point=BitFlippingEnv,
)

env = DummyVecEnv([lambda: gym.make("BitFlippingEnv-v1")]*4)
env = VecMonitor(env, info_keywords=("is_success",), filename="extra_info_vec")

model = PPO("MultiInputPolicy", env, verbose=1)
model.learn(total_timesteps=5000)
```

## Checklist:
- [x] I've read the [CONTRIBUTION](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) guide (**required**)
- [x] I have updated the changelog accordingly (**required**).
- [ ] My change requires a change to the documentation.
- [x] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [ ] I have updated the documentation accordingly.
- [x] I have reformatted the code using `make format` (**required**)
- [x] I have checked the codestyle using `make check-codestyle` and `make lint` (**required**)
- [x] I have ensured `make pytest` and `make type` both pass. (**required**)
- [x] I have checked that the documentation builds using `make doc` (**required**)

### CODE REVIEW

**Comment by araffin** [comment]
Hello,
thanks for the pr, please add a test and we are good to go ;)

**Comment by ScheiklP** [comment]
Huh, that's weird... the codestyle check runs smooth on my end

```❯ make check-codestyle
# Sort imports
isort --check stable_baselines3/ tests/ docs/conf.py setup.py
# Reformat using black
black --check -l 127 stable_baselines3/ tests/ docs/conf.py setup.py
All done! ✨ 🍰 ✨
92 files would be left unchanged.
```

**Comment by araffin** [comment]
> Huh, that's weird... the codestyle check runs smooth on my end

Do you have latest version of black and isort?

File: tests/test_vec_monitor.py [hunk 1]
```diff
@@ -45,6 +49,41 @@ def test_vec_monitor(tmp_path):
...
+    """
+    Test loggig `info_keywords` in the `VecMonitor` wrapper
+    """
+    monitor_file = os.path.join(str(tmp_path), f"stable_baselines-test-{uuid.uuid4()}.monitor.csv")
+
+    register(
+        id="BitFlippingEnv-v1",
+        entry_point=BitFlippingEnv,
+    )
+    env = DummyVecEnv([lambda: gym.make("BitFlippingEnv-v1")])
```
Comment (araffin) [inline tests/test_vec_monitor.py:62]: please use `make_vec_env` and you can pass the `BitFlippingEnv` as argument, you don't need to register the env

**Comment by ScheiklP** [comment]
> 

>>> black.__version__
'21.12b0'
>>> isort.__version__
'5.10.1'

**Review by araffin — APPROVED** [review]
LGTM, thanks =)

**Pull request merged**